### ISSUE TITLE
Emacs org-mode cheatsheet

### ISSUE THREAD

**Issue by lemorage** [LeCoupa/awesome-cheatsheets#378]
Will there be a plan to add an excellent org-mode cheatsheet, with very clear and concise comments? I think it is very helpful to grasp org-mode in our workflows in the process of making this cheatsheet.

**Comment by Madhusri02** [comment]
It is sounds good in bringing up org-mode cheatsheet . It would be a valuable resource to the user when using the org-mode cheatsheet. I'm willing to contribute to the creation of such a cheatsheet . Looking forward to hearing your thoughts on this!

**Comment by lemorage** [comment]
@Madhusri02 Yes, I agree with you! I can't assign you, but you can fork this repo and make a PR directly. Good luck!
